### DISCUSSION TITLE
Edge Function Crashes on Deployment to Supabase, Works Locally

### DISCUSSION THREAD

**Discussion by rlataguerra** [supabase/supabase#20342] [Questions]
I'm experiencing a problem with deploying an edge function to Supabase. 
The function runs without issues when invoked locally using the Supabase CLI or Deno, but it crashes immediately upon deployment to Supabase.

Here is the code, I removed everything, just for debugging and it still crash, without any information :

```
Deno.serve((_req) => {
  return new Response('ok', {
    headers: {
      'Access-Control-Allow-Origin': '*',
      'Access-Control-Allow-Methods': 'GET, POST, OPTIONS',
      'Access-Control-Allow-Headers': 'authorization, x-client-info, apikey, content-type'
    },
    status: 200
  });
});
```

I tried to deploy with and without --legacy-bundle

**Comment by awalias** [comment]
>crashes immediately upon deployment

hey @rlataguerra could you share any error messages?

**Comment by mansueli** [comment]
I've deployed this function here and it is working. Are you calling it with the JWT?  

What is the CLI version that you are using?

**Comment by laktek** [comment]
@rlataguerra If you're deploying the functions to Supabase, you can use the logs in Supabase Dashboard to see what's causing the crashes/errors. Check this guide to learn how to use logs and other ways to debug Edge Functions issues https://supabase.com/docs/guides/functions/debugging#hosted

If you still have issues, feel free to open a support ticket (https://supabase.help), so we can look into your project logs from our end.

**Comment by rlataguerra** [comment]
Hi @awalias !

After a quick email with the support, their intuition is a problem with one of your reverse proxy. For now everything is back to normal 🙏
